docs/admin/daemons.md

```diff
@@ -216,6 +216,10 @@ DaemonSet is not enabled by default. Enable it by setting
 achieved by exporting KUBE_ENABLE_DAEMONSETS=true before running kube-up.sh script
 on GCE.
 
+If you intend to restart the api server in an existing cluster to enable this
+feature, you will also need to restart the controller manager
+(`kube-controller-manager`) to ensure that DaemonSet controllers can be created.
+
```

Comment:
Can we put a little more details on why this is required? Does the controller manager read specific settings from the API Server at startup?